src/v3/src/components/Form/Form.tsx

```diff
@@ -11,45 +11,18 @@
  */
 
 import { IdxMessage } from '@okta/okta-auth-js';
-import { clone } from 'lodash';
+import clone from 'lodash/clone';
 import { FunctionComponent, h } from 'preact';
 
 import { useWidgetContext } from '../../contexts';
 import { useOnSubmit } from '../../hooks';
 import {
-  ButtonElement,
-  ButtonType,
-  isUISchemaLayoutType,
   SubmitEvent,
   UISchemaLayout,
-  UISchemaLayoutType,
 } from '../../types';
-import { resetMessagesToInputs } from '../../util';
+import { getSubmitButtonSchema, resetMessagesToInputs } from '../../util';
 import { renderUISchemaLayout } from './renderUISchemaLayout';
 
-// TODO: remove this function once submission related meta is in schema
-const getSubmitButtonSchema: any = (uischema: UISchemaLayout, stepperStepIndex: number) => {
```

Comment:
please keep it here to avoid conflict, I handled/removed it in my coming PR

Reply:
The hook I created needs it too, what should I do in that case?

Reply:
If yours can be merged first I don't mind accepting changes